steps/server11.js

```diff
@@ -0,0 +1,27 @@
+// After open the localhost:3000 in the browser
```

Comment:
It might not be possible to see the server-side logs when this is in production. Either way, I tried to simplify this a bit:

```
// The callback you wrote actually receives one argument -- the socket that triggered the event.
// Pass `socket` as a parameter to the callback function you just wrote and log `Socket connected: <id of the socket>` to the console instead of "User connected".
```